liblocation/src/main/java/com/mapbox/android/core/location/GoogleLocationEngineImpl.java

```diff
@@ -46,7 +46,9 @@ public void getLastLocation(@NonNull LocationEngineCallback<LocationEngineResult
     throws SecurityException {
     GoogleLastLocationEngineCallbackTransport transport =
       new GoogleLastLocationEngineCallbackTransport(callback);
-    fusedLocationProviderClient.getLastLocation().addOnSuccessListener(transport).addOnFailureListener(transport);
+    Task<Location> lastLocation = fusedLocationProviderClient.getLastLocation();
```

Comment:
i'm still a bit confused about this change, what difference does it make? the first call into `addOnSuccessListener(transport)` passes through instance of `Task<Location>` making this chain call possible.